subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http/HttpServer.groovy

```diff
@@ -625,6 +628,18 @@ class HttpServer extends ServerWithExpectations implements HttpServerFixture {
         collection.addHandler(handler)
     }
 
+    /**
+     * Blocks on SSL handshake for 60 seconds.
+     */
+    void expectSslHandshakeBlocking() {
+        sslPreHandler = new BiAction<EndPoint, Request>() {
+            @Override
+            void execute(EndPoint endPoint, Request request) {
+                Thread.sleep(TimeUnit.SECONDS.toMillis(60))
```

Comment:
:o: Maybe use CountDownLatch here like in `blocking()`?

Reply:
Yes, I did notice there's a usage of `CoundDownLatch` in this class, but could you tell me why we should use that instead of `Thread.sleep`?

Reply:
I guess in terms of functionality there's no real difference. At the moment I have no argument for or against one of them.

Reply:
Fine, I prefer `Thread.sleep` because `CountDownLatch` is more verbose: https://github.com/gradle/gradle/blob/9223486d4d2cd039d5dd7d408ea06796d73c7413/subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http/HttpServer.groovy#L281